indico/migrations/versions/20210715_1211_9d00917b2fa8_add_event_category_request.py

```diff
@@ -0,0 +1,64 @@
+"""Add event category request
+
+Revision ID: 9d00917b2fa8
+Revises: 1cec32e42f65
+Create Date: 2021-07-15 12:11:28.157824
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+from indico.core.db.sqlalchemy import PyIntEnum, UTCDateTime
+# revision identifiers, used by Alembic.
+from indico.modules.categories.models.event_move_request import MoveRequestState
```

Comment:
Yes, I saw some other cases where the Enum was manually imported so I assumed this was something to avoid from the auto-generated revision. But that's a good point, I can restore the original.

Reply:
It's actually custom code somewhere in `indico/core/db/` that generates the embedded enums ;)